components/library/libgtop/patches/libgtop-01-illumos.patch

```diff
@@ -0,0 +1,1765 @@
+diff -ru --new-file libgtop-2.40.0/aclocal.m4 libgtop-2.40.0.new/aclocal.m4
```

Comment:
Avoid patching generated files, if possible.

Reply:
Ok need to find a way to run aclocal first as otherwise the build blows up.

Reply:
Usually this looks like 
`COMPONENT_PREP_ACTION = (cd $(@D) && autoreconf -fi)`